src/libs/ReportActionComposeFocusManager.ts

```diff
@@ -60,6 +63,7 @@ function clear(isPriorityCallback = false) {
     if (isPriorityCallback) {
         priorityFocusCallback = null;
     } else {
+        editComposerRef.current = null;
```

Comment:
Don't we have to reset `editComposerRef` when `isPriorityCallback` is true because we call `clear(true)` in `ReportActionItemMessageEdit`?